cmd/pbnj/server.go

```diff
@@ -48,7 +52,11 @@ var (
 			grpc_zap.ReplaceGrpcLoggerV2(zlog)
 
 			grpcServer := grpc.NewServer(
+				grpc_middleware.WithStreamServerChain(
+					grpc_prometheus.StreamServerInterceptor,
+				),
```

Comment:
I don't think we're supporting a streaming server at the moment

Reply:
Yeah, probably not then. Will remove that.